coordinator/src/db/positions.rs

```diff
@@ -67,6 +67,24 @@ impl Position {
         Ok(positions)
     }
 
+    pub fn get_all_positions_with_expiry_within(
+        conn: &mut PgConnection,
+        start: OffsetDateTime,
+        end: OffsetDateTime,
+    ) -> QueryResult<Vec<crate::position::models::Position>> {
+        let positions = positions::table
+            .filter(positions::expiry_timestamp.gt(start))
+            .filter(positions::expiry_timestamp.lt(end))
```

Comment:
Just to confirm, you are using here `>` and not `>=`, so explicitly exclusive the boundaries.